citystreets/settings.py

```diff
@@ -174,3 +174,10 @@
     "MIN_ZOOM": 3,
     "MAX_ZOOM": 18,
 }
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
```

Comment:
did you test to make sure that account confirmation and password reset emails were still being sent out correctly?

Reply:
Yes, just double-checked that! Oh, I didn't realize you had a gmail account already setup. Maybe could have used that instead.

Reply:
oh yeah, that'd probably be a good idea if it works. It would be weird to have the site sending out emails from 2 different email accounts.

Reply:
We would need to specify that email address and password in the `settings.py` file. Currently, it sends all emails (landlord contact, email authentication, password reset) from the same email address (fivefries8@gmail.com).

Reply:
the email settings for allauth are on lines 88-92 in `settings.py`. You could just copy from there.

Reply:
the email settings for allauth are on lines 88-92 in `settings.py`. You could just copy from there.

Reply:
Perfect! Didn't see that before. It now uses "team.five.fries@gmail.com" for all the emails